src/web/jdv/client/src/KMZExport.ts

```diff
@@ -37,8 +38,13 @@ async function taskPacketToKMLPlacemarks(taskPacket: LogTaskPacket) {
     const dateString = formatter.format(date)
     const bot_id = taskPacket.bot_id
 
-    const diveIconUrl = 'file://files/diveIcon.png'
-    const driftArrowHeadUrl = 'file://files/driftArrowHead.png'
+    // We omit the file:// here, so that the KMZ can be opened properly in Google Earth
+    const diveIconUrl = 'files/diveIcon.png'
+
+    function getDriftArrowHeadUrl(drift: DriftPacket) {
```

Comment:
Can we add the a comment before this function like the driftspeedtobinindex function  /***/